ThinBridge/Chronos
# Settings dialog: Use Left/Up/Right/Down instead of arrow glyph

In the previous version, arrow glyph is used to represent key combination of unlocking restriction.
This causes MOJIBAKE when display language is english.

To resolve such situation, we use Left/Up/Right/Down explicitly in english mode.

# Which issue(s) this PR fixes:

N/A


# What this PR does / why we need it:

In the previous version, arrow glyph is used to represent key combination of unlocking restriction.
This causes MOJIBAKE when display language is english.

To resolve such situation, we use Left/Up/Right/Down explicitly in english mode.

# How to verify the fixed issue:

## The steps to verify:

1. Launch Chronos in english mode 
2. Go to Tools > Preferences
3. Check Restrict access to options group configuration

## Expected result:

Key combination checkbox must be "Shift, Ctrl, Alt, Left, Up, Right, Down".
